linux_os/guide/system/bootloader-grub2/grub2_nosmap_argument_absent/rule.yml

```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+title: 'Ensure SMAP is not disabled during boot'
+
+description: |-
+    The SMAP is used to prevent the supervisor mode from unintentionally reading/writing into
+    memory pages in the user space, it is enabled by default since Linux kernel 3.7.
+    But it could be disabled through kernel boot parameters.
+
+    Ensure that Supervisor Mode Access Prevention (SMAP) is not disable by
```

Comment:
```suggestion
    Ensure that Supervisor Mode Access Prevention (SMAP) is not disabled by
```

Reply:
Fixed by rebase in 4cc98d2bd6098f214cd4c5af012657067ae7378c